hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -1762,20 +1762,15 @@ public List<OzoneKey> listKeys(String volumeName, String bucketName,
   @Override
   public List<RepeatedOmKeyInfo> listTrash(String volumeName, String bucketName,
       String startKeyName, String keyPrefix, int maxKeys) throws IOException {
-
-    Preconditions.checkNotNull(volumeName);
-    Preconditions.checkNotNull(bucketName);
-
-    return ozoneManagerClient.listTrash(volumeName, bucketName, startKeyName,
-        keyPrefix, maxKeys);
+    // listTrash is deprecated
+    throw new UnsupportedOperationException();
```

Comment:
Should we mark the methods with a `@Deprecated` annotation instead of removing their implementation and throwing an `UnsupportedOperationException`, for the sake of consistency 

Perhaps similar to `setEncryptionKey()` and `renameKeys()` APIs (now deprecated)?

Reply:
Updated in the latest commit [b645e54](https://github.com/apache/ozone/pull/7060/commits/b645e5448bf06a866dfff844e640a6bd64053bf9)